ext/nnstreamer/tensor_filter/tensor_filter_python3.hh

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
```

Comment:
You don't need this header file. You can declare the class in .cc file unless you have external users that do ```#include <tensor_filter_python3.hh>```.

Reply:
Oh I didn't know that.
removed header file!
Thank you 👍